sbol/property.py

```diff
@@ -697,17 +697,25 @@ def get(self, item):
         return reference_store[item]
 
     def set(self, uri):
+        # TODO for parity with Property, this should perhaps be split into calls to two 
+        # subordinate methods for setSinglePropertyValue and setPropertyValueList
```

Comment:
You're using an old copy of property.py here. ReferencedObject.set was broken out to setSinglePropertyValue and setPropertyValueList in mid-February.

Reply:
Fixed